### PR TITLE
Introduce Versatile Express FVP platform to arm-trusted-firmware.

### PR DESCRIPTION
**Pull request by uarif1**
This platform port includes support for Cortex-A7 and A5 models of FVP Versatile express

### CODE REVIEW

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Comment by antonio-nino-diaz-arm** [comment]
There are checkpatch.pl issues. Fix all ERRORs.

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

File: include/lib/xlat_tables/xlat_tables_defs.h [hunk 1]
```diff
@@ -73,7 +73,11 @@
 #define PAGE_SIZE_MASK		(PAGE_SIZE - U(1))
 #define IS_PAGE_ALIGNED(addr)	(((addr) & PAGE_SIZE_MASK) == U(0))
 
-#define XLAT_ENTRY_SIZE_SHIFT	U(3) /* Each MMU table entry is 8 bytes (1 << 3) */
+#if ARM_ARCH_MAJOR == 7 && !ARMV7_SUPPORTS_LARGE_PAGE_ADDRESSING
```
Comment (antonio-nino-diaz-arm) [inline include/lib/xlat_tables/xlat_tables_defs.h]: The only remaining CI failure is a MISRA issue, you can fix it by changing this to:
```
#if (ARM_ARCH_MAJOR == 7) && !ARMV7_SUPPORTS_LARGE_PAGE_ADDRESSING
```

Comment (uarif1) [reply include/lib/xlat_tables/xlat_tables_defs.h]: Thanks, Done.

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Pull request merged**